src/app/services/actions/permission-mode-action-creator.ts

```diff
@@ -0,0 +1,11 @@
+import { IAction } from '../../../types/action';
```

Comment:
Typescript allows to config for a non-absolute way for importing modules if you feel there're too many `../../` 

https://www.typescriptlang.org/tsconfig#baseUrl

Reply:
i think all the action creators have it imported as `import { IAction } from '../../../types/action';` so i'll have to ask the graph explorer maintainers if they would want to change the base url.